### ISSUE TITLE
The character "里" is missing from the traditional Chinese pinyin IME.

### ISSUE THREAD

**Issue by patrickdark** [fcitx/fcitx#420]
As the title indicates, the character "里" is missing from the traditional Chinese pinyin IME. This means that there's no way to type words like "公里".

**Comment by wengxt** [comment]
Which input method? I don't quite get it. Pinyin + Traditional chinese conversion or?

**Comment by patrickdark** [comment]
@wengxt Yes, in Linux Mint 18.3, the "Pinyin" input method with "Traditional Chinese" enabled.

**Comment by patrickdark** [comment]
![screenshot from 2018-06-03 00-04-36 cropped](https://user-images.githubusercontent.com/10335392/40883447-d43067e4-66c2-11e8-81c1-47d5b5c3e4a0.png)

**Comment by wengxt** [comment]
I'd say it works as intended. The "Simp/Trad" feature you see, is based on conversion. The underlying data and algorithm are all based on the input method you're using. It is not designed for typing Traditional Chinese seriously, but just to help you type something when you need it urgently or you have little knowledge on the Traditional Chinese. 

You should use a real traditional chinese input method if you want to get the full support. Currently, options include Canjie, Chewing etc. If you need Pinyin you may want to try fcitx-libpinyin and check the use Traditional Chinese data box in configuration.

Or, I think sunpinyin allows you to type a larger charset for Chinse.

**Comment by patrickdark** [comment]
@wengxt The screenshot in my previous post shows fcitx-pinyin with the custom fcitx skin found in Linux Mint. I installed fcitx-libpinyin, but it appears to be the same software as fcitx-pinyin.

I tried fcitx-sunpinyin. The result is the same as fcitx-libpinyin: I can't type "公里" with the "Traditional Chinese" option enabled.

I already had Chewing installed (via fcitx-pinyin) to practice using Bopomofo. Neither the "Default Keyboard" (Bopomofo) nor "Han-Yu PinYin Keyboard" configurations allow me to type "公里".

fcitx-table-cangjie-big works, but temporarily switching to it reduces my typing speed.

It seems that the fastest workaround is to type Shift+Ctrl+U→91cc.
